src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import { DataAttributes } from "../../helpers/Attributes";
+
+export interface TileProps extends React.AriaAttributes, DataAttributes {
+  /**
+   * A href link address that is passed into the Tile component
+   */
+  link: string;
+  /**
+   * A title wrapped in an anchor tag.
+   */
+  title: string | null;
```

Comment:
This and the email should be listed as `string | undefined`, not null.
Also, you should be able to just mark this as `title?: string`; as by marking it as not required, it can implicitly be undefined.

Reply:
Great, will change this thanks